apps/calendar/build/calendar.build.js

```diff
@@ -17,15 +18,34 @@
 
   modules: [
     {
-      name: 'ext/alameda',
+      create: true,
```

Comment:
`create` means that it creates a new module/file named `bundle`.. decided to use this name instead `ext/curl` since it makes more sense.